bash/maven_build.sh

```diff
@@ -45,7 +45,10 @@ printUsage() {
 
 checkforMavenProblems() {
 	echo "Check for Maven Problems on Product:"
-	(grep -n "\[\(WARN\|WARNING\|ERROR\)\]" maven.log | grep -v "\[WARNING\] Checksum validation failed" || exit 0  && exit 1;)
```

Comment:
This one should have been handled by the PR #264. Please check that there is no LF/CR issue here. There should be no difference to Development.